bin/MadGraph5_aMCatNLO/cards/WZ_leptonic/WLLjj_dim8_ewk_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set complex_mass_scheme False
+import model SM_Ltotal_Ind5v2020v2_UFO
+
+define p = g u c d s u~ c~ d~ s~
+define j = p b b~
```

Comment:
Is there a reason to define p without bottom for WZ but with bottom for WW?
Why are the protons differently defined?

Reply:
Hi @sihyunjeon 
In this case, the WZb is considered background, we have another "background" tZq sample, that's why the b decays are removed in the "signal"

Reply:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/17442656/124571554-d96f5900-de82-11eb-8e8a-2e38841b8a33.png">

There are diagrams like this. You mean you consider such diagrams as background and consider it from there?

Reply:
Hi @sihyunjeon We consider WZbq as a background in our analysis.